Ensure pipeline description does not omit anything from main full pipeline doc (#110)
Imageomics/kabr-tools: docs/pipeline/preprocessing.md

## Resources

> - See the Wiki [CVAT User Guide](https://github.com/Imageomics/kabr-tools/wiki/CVAT-User-Guide) and [Data Management Tips](https://github.com/Imageomics/kabr-tools/wiki/Data-Management-Tips) for detailed instructions and recommendations.

Reviewer: ```suggestion
- See our CVAT [User Guide](cvat/cvat-guide.md) and [Data Management Tips](cvat/cvat-data-management.md) for detailed instructions and recommendations.
```
